.github/workflows/main.yml

```diff
@@ -0,0 +1,20 @@
+name: Format Files
+on:
+  pull_request:
+    branches: [main]
+jobs:
+  format:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.head_ref }}
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '14.x'
```

Comment:
Node 14 is already a year past the end of active support and will soon cease security support: https://endoflife.date/nodejs

Let's use Node 18 here if possible, or at least 16